soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3330,24 +3342,137 @@ void DrawRandoEditor(bool& open) {
+                            "\n"
+                            "Greg as Wildcard - Greg does not change logic, Greg helps open the bridge, max number of "
+                            "rewards on slider does not change.");
+
+                        UIWidgets::EnhancementCombobox("gRandomizeRewardOptions", randoRewardOptions, RO_REWARD_MAX, RO_REWARD_STANDARD);
+                        switch (CVarGetInteger("gRandomizeRewardOptions", RO_REWARD_STANDARD)) {
+                            case RO_REWARD_STANDARD:
+                                ImGui::Dummy(ImVec2(0.0f, 0.0f));
+                                UIWidgets::EnhancementSliderInt("Stone Count: %d", "##RandoStoneCount",
+                                                                "gRandomizeStoneCount", 1, 3, "", 3, true);
+                                break;
+                            case RO_REWARD_GREG:
+                                ImGui::Dummy(ImVec2(0.0f, 0.0f));
+                                UIWidgets::EnhancementSliderInt("Stone Count: %d", "##RandoStoneCount",
+                                                                "gRandomizeStoneCount", 1, 4, "", 4, true);
+                                break;
+                            case RO_REWARD_WILDCARD:
+                                ImGui::Dummy(ImVec2(0.0f, 0.0f));
+                                UIWidgets::EnhancementSliderInt("Stone Count: %d", "##RandoStoneCount",
+                                                                "gRandomizeStoneCount", 1, 3, "", 3, true);
+                                break;
+                        }
+
                         break;
                     case RO_BRIDGE_MEDALLIONS:
-                        ImGui::Dummy(ImVec2(0.0f, 0.0f));
-                        UIWidgets::EnhancementSliderInt("Medallion Count: %d", "##RandoMedallionCount",
-                                                        "gRandomizeMedallionCount", 1, 6, "", 6, true);
+                        ImGui::Text("Reward Options");
+                        UIWidgets::InsertHelpHoverText(
+                            "Standard Rewards - Greg does not change logic, Greg does not help open the bridge, max "
+                            "number of rewards on slider does not change.\n"
+                            "\n"
+                            "Greg as Reward - Greg does change logic (can be part of expected path for opening "
+                            "bridge), Greg helps open bridge, max number of rewards on slider increases by 1 to "
+                            "account for Greg. \n"
+                            "\n"
+                            "Greg as Wildcard - Greg does not change logic, Greg helps open the bridge, max number of "
+                            "rewards on slider does not change.");
+
+                        UIWidgets::EnhancementCombobox("gRandomizeRewardOptions", randoRewardOptions, RO_REWARD_MAX, RO_REWARD_STANDARD);
```

Comment:
That also means you don't need the MAX option in your enum at all, and you'll see that I've also taken out all the MAX options in my PR.